Apache RAT (license audit...) complains for some reason about location.o,
 causing maven to fail. This patch is a "dirty" workaround.

--- a/pom.xml
+++ b/pom.xml
@@ -140,6 +140,8 @@
               <exclude>src/native/unix/config.nice</exclude>
               <exclude>src/native/unix/config.status</exclude>
               <exclude>src/native/unix/config.log</exclude>
+              <!-- Workaround for location.o -->
+              <exclude>src/native/unix/native/location.o</exclude>
             </excludes>
           </configuration>
         </plugin>
